src/index.js

```diff
@@ -20,6 +20,7 @@ const env = process.env.NODE_ENV
 const port = process.env.PORT
 const certPassphrase = process.env.CERT_PASSPHRASE
 const balancerNetwork = process.env.BALANCER_NETWORK
+const uniswapNetwork = process.env.UNISWAP_NETWORK
```

Comment:
For now, we can use the `BALANCER_NETWORK` field, which should be called `ETHEREUM_NETWORK` in the future.